doc/api/repl.md

```diff
@@ -38,6 +38,7 @@ The following special commands are supported by all REPL instances:
   `> .save ./file/to/save.js`
 * `.load` - Load a file into the current REPL session.
   `> .load ./file/to/load.js`
+* `.paste` - Enter paste mode (`<ctrl>-D` to finish, `<ctrl>-C` to cancel)
```

Comment:
@Fishrock123 
Instead of going back and forth fixing description message, if anyone suggest a precise description, Its easy for me to update the PR 😄 

[This](https://github.com/nodejs/node/pull/7275#issuecomment-226807237) comment may help.